test/functional/test_ssh_server_comes_up_for_functional_tests.rb

```diff
@@ -1,24 +0,0 @@
-require 'helper'
-
-module SSHKit
-
-  class TestHost < FunctionalTest
-
-    def host
-      @_host ||= Host.new('')
-    end
-
-    def test_that_it_works
-      assert true
-    end
-
-    def test_creating_a_user_gives_us_back_his_private_key_as_a_string
```

Comment:
🗒️ This seemed to be testing the Vagrant infrastructure itself, as opposed to testing functionality of SSHKit. It doesn't serve a purpose with Docker, so I removed it.